src/Cake.Common/Tools/XUnit/XUnit2Aliases.cs

```diff
@@ -19,6 +19,12 @@ public static class XUnit2Aliases
         /// </summary>
         /// <param name="context">The context.</param>
         /// <param name="pattern">The pattern.</param>
+        /// <example>
+        /// <code>
+        /// var configuraton = "Release";
+        /// XUnit2("./src/**/bin/" + configuraton + "/*.Tests.dll");
```

Comment:
I think this could be simplified even more. Suggestion is to remove the `configuration` variable.